### DISCUSSION TITLE
Change color for classic layer OSM

### DISCUSSION THREAD

**Discussion by maxk1m** [Mapsui/Mapsui#1850] [Q&A]
I want to change the Style property of a type TileLayer, but my ThemeStyle property doesn't work. Where i can look examples, how to change color polygons, lines, etc. ?

**Comment by pauldendulk** [comment]
That is not possible since the osm layer used in Mapsui is using image tiles which are generated by the server.

**Reply by maxk1m** [reply]
Thanks for the answer. At least I will not look now in this direction of the decision. In general, I need a layer for a dark theme. And I wanted to take umap osm maps, but no matter how I tried to write a url to get a png image, nothing came of it. And I couldn't find any information on how to do it. I mean maps from umap.openstreetmap.fr
